brvs-core/src/main/java/iroha/validation/transactions/provider/impl/BasicTransactionProvider.java

```diff
@@ -43,7 +43,7 @@
   private final BrvsIrohaChainListener irohaReliableChainListener;
   private final ScheduledExecutorService executorService = Executors.newScheduledThreadPool(3);
   private final Set<String> userDomains;
-  private boolean isStarted;
+  private volatile boolean isStarted;
```

Comment:
No need to use volatile, because you read and modify this field in `synchronized` block